fut/src/App.js

```diff
@@ -3,11 +3,33 @@ import './App.css';
 import Topbar from './components/Topbar';
 
 class App extends Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      input : '',
+    }
+    this.onInputChange = this.onInputChange.bind(this);
```

Comment:
comme tu as défini `onInputChange` comme une arrow function, `this` dans `onInputChange` refer au "outer context" qui est le `this` de la class `App`. Donc tu peux referencer `this.props` ou `this.state` ou n'importe quelle autre method que tu as défini sur `App` sans faire ca `this.onInputChange.bind(this);` 

le call `.bind(this)` retourne une nouvelle fonction dont le context (`this`) est ce qui est passé entre les parentheses. Mais plus besoin de faire ca avec les arrow function pack elles choppent automatiquement le outer `this`.